app/controllers/admin/payments_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Admin::PaymentsController < ApplicationController
+  before_filter :require_logged_in
+  before_filter :require_admin
+
+  def index
+    @payments = Payment.all
```

Comment:
You may want to consider using pagination here as well. Showing all Payments for admin might take a lot of time to load.